store/sqlstore/team_store.go

```diff
@@ -374,8 +378,14 @@ func (s SqlTeamStore) SearchAll(term string) ([]*model.Team, *model.AppError) {
 	var teams []*model.Team
 
 	term = sanitizeSearchTerm(term, "\\")
+	term = wildcardSearchTerm(term)
 
-	if _, err := s.GetReplica().Select(&teams, "SELECT * FROM Teams WHERE Name LIKE :Term OR DisplayName LIKE :Term", map[string]interface{}{"Term": term + "%"}); err != nil {
+	operatorKeyword := "ILIKE"
```

Comment:
I'd rather that we do not use Postgres (or any SQL DB provider) SQL extensions and stick as much as possible to vanilla SQL, otherwise porting the SQL code to any other provider (e.g. MSSQL) becomes an even more difficult task. Also, we should use/factor out `buildLIKEClause` and `buildFulltextClause` for both channels and teams

Reply:
Ticket created to keep track https://mattermost.atlassian.net/browse/MM-26727